doc/classes/Parallax2D.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Parallax2D" inherits="Node2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A node used to create a parallax scrolling background.
+	</brief_description>
+	<description>
+		A Parallax2D is used to create a parallax effect. It can move at a different speed relative to the camera movement using [member scroll_scale]. This creates an illusion of depth in a 2D game. If manual scrolling is desired, the [Camera2D] position can be ignored with [member override_ignore_camera_scroll].
+		[b]Note:[/b] Any changes to this node's position made after it enters the scene will be ignored.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="autoscroll" type="Vector2" setter="set_autoscroll" getter="get_autoscroll" default="Vector2(0, 0)">
+			When set, the offset will scroll automatically by the set amount of pixels per second.
+		</member>
+		<member name="limit_begin" type="Vector2" setter="set_limit_begin" getter="get_limit_begin" default="Vector2(0, 0)">
+			Top-left limits for scrolling to begin. If the camera is outside of this limit, the Parallax2D will stop scrolling. Must be lower than [member limit_end] minus the viewport size to work.
+		</member>
+		<member name="limit_end" type="Vector2" setter="set_limit_end" getter="get_limit_end" default="Vector2(0, 0)">
+			Bottom-right limits for scrolling to end. If the camera is outside of this limit, the Parallax2D will stop scrolling. Must be higher than [member limit_begin] and the viewport size combined to work.
+		</member>
+		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
+			The Parallax2D's offset relative to its parent.
+		</member>
+		<member name="override_follow_viewport" type="bool" setter="set_follow_viewport" getter="get_follow_viewport" default="true">
+			If [code]true[/code], Parllax2D will be offset by the current camera's position. It may be desired to turn off if the Parallax2D is in a [CanvasLayer] separate from the current camera.
```

Comment:
Slightly simplified.
```suggestion
			If [code]true[/code], this [Parallax2D] is offset by the current camera's position. It may be desired to disable it if the [Parallax2D] is in a [CanvasLayer] separate from the current camera.
```